translator/config/schema.json

```diff
@@ -987,6 +1010,9 @@
             "http_endpoint": {
               "description": "HTTP endpoint to use to listen for OTLP JSON traces",
               "$ref": "#/definitions/endpointOverrideDefinition"
+            },
+            "tls": {
```

Comment:
So this is under `otlpDefinition` which is for `otlp` as per [this](https://github.com/aws/amazon-cloudwatch-agent/pull/1028/files#diff-b316c7f1304a4a8f2e1707d12dce4eebd0d9cf1c9c3add74cf31b01bfcdd5299R939). How does it work for your when its under `app_signals` thats [here](https://github.com/aws/amazon-cloudwatch-agent/pull/1028/files#diff-b316c7f1304a4a8f2e1707d12dce4eebd0d9cf1c9c3add74cf31b01bfcdd5299R930)?

Reply:
Doesn't app_signals inherit otlp definitions? I may have misunderstood that, if thats not the case I can move the tls reference to under app_signals

Reply:
Can you check why none of your testing or unit tests failed with this bug?
And can you add a test to prevent that from happening?